ci/jenkins/test-vm.sh

```diff
@@ -143,10 +143,16 @@ function apply_antrea {
     fi
     TEMP_ANTREA_TAR="antrea-image.tar"
     docker save antrea/antrea-agent-ubuntu:latest antrea/antrea-controller-ubuntu:latest -o $TEMP_ANTREA_TAR
-    ctr -n k8s.io image import $TEMP_ANTREA_TAR
+    kubectl get nodes --selector=kubernetes.io/os=linux --no-headers=true -o custom-columns=IP:.status.addresses[0].address | while read -r IP; do
+        rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no" $TEMP_ANTREA_TAR jenkins@${IP}:${WORKDIR}/$TEMP_ANTREA_TAR
+        ssh -o StrictHostKeyChecking=no -n jenkins@${IP} "crictl rmi --prune; crictl ps --state Exited; ctr -n=k8s.io images import ${WORKDIR}/$TEMP_ANTREA_TAR" || true
```

Comment:
out of curiosity, why didn't we have that step (loading the image to the Nodes) before?

Reply:
Previously, both Docker and containerd were installed on the same testbed. To avoid potential compatibility issues after kubernetes removed docker runtime, we have implemented the new topology for all jenkins testbed: one node (the jumper node) has a Jenkins agent and Docker installed, while another node has containerd and Kubernetes installed. When running CI, we need to copy the image from the jumper node to the Kubernetes node.